modules/sharedaddy/sharing-sources.php

```diff
@@ -623,20 +623,28 @@ public function get_display( $post ) {
 		$via = $this->sharing_twitter_via( $post );
 
 		if ( $via ) {
-			$via = '&via=' . rawurlencode( $via );
+			$via = 'data-via="' . $via . '"';
```

Comment:
To be on the safe side, we should probably keep some sanitization here, even though `sharing_twitter_via` removes invalid characters already on our behalf. This could change in the future, so perhaps adding an `esc_attr` is the best thing here.